injected/scripts/utils/build.js

```diff
@@ -48,7 +50,7 @@ export async function bundle(params) {
             'import.meta.injectName': JSON.stringify(platform),
             'import.meta.trackerLookup': trackerLookup,
         },
-        plugins: [loadFeaturesPlugin],
+        plugins: [loadFeaturesPlugin, commentPlugin({ pathMatch: 'seedrandom/lib/alea', regex: /^\/\/ Copyright \(C\)/ })],
```

Comment:
Can we just match all files and check for any `/copyright/i` as a string in comments?